src/components/Menu/Menu.style.ts

```diff
@@ -1,29 +1,84 @@
-import type { Theme } from '../../theme';
-import type { RequiredProperties } from '../../utils/common';
-import type { ButtonProps } from '../Button/Button';
-
-const getButtonMargin =
-  ({ iconLeftName, iconRightName }: Pick<ButtonProps, 'iconLeftName' | 'iconRightName'>) =>
-  (theme: Theme) => {
-    if (iconLeftName || iconRightName) {
-      return theme.globals.spacing.get('6');
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { rem } from 'polished';
+import { Menu, MenuItem } from 'react-aria-components';
+
+import { getMenuTokens } from './Menu.tokens';
+import type { ListRowSize } from '../List';
+import { getListItemTokens } from '../List/List.tokens';
+import { LIST_ITEM_HEIGHT } from '../List/utils';
+import { body02, body03, label02, label03 } from '../Typography/Typography.config.styles';
+
+export const MenuItemWrapper = styled(MenuItem)<{
+  isDisabled?: boolean;
+  isCompact?: boolean;
+  rowSize?: ListRowSize;
+}>(({ rowSize, isCompact, isDisabled, theme }) => {
+  const listItemTokens = getListItemTokens(theme);
+  const height = rem(LIST_ITEM_HEIGHT[rowSize]);
+  const padding = css`0 ${listItemTokens('paddingHorizontal')}`;
+  const itemTypographyStyle = isCompact ? body03(theme) : body02(theme);
```

Comment:
are there menu tokens for these maybe?

Reply:
MenuItemWrapper is just a wrapper of listitems, these are the tokens for them